sci-geosciences/josm-bin/josm-bin-17428.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils java-utils-2
+
+DESCRIPTION="Java-based editor for the OpenStreetMap project"
+HOMEPAGE="https://josm.openstreetmap.de/"
+
+MY_PV="snapshot-${PV}"
+[[ ${PV} == "99999" ]] && MY_PV="latest"
```

Comment:
I image this will be out of sync with the manifest all the time

Reply:
I think it is already

Reply:
I am not sure i understand the comment, please resolve the conversation if it was not important, or go into detail

Reply:
Using a SRC_URI pointing towards `josm-latest.jar` will generate a manifest for it. As `josm-latest.jar` has already been updated on the upstream sider, the checksum in the manifest file is already incorrect. You can't rely on a continually updating file within SRC_URI for this reason

Reply:
Yes, thanks. I will push an ebuild that renames it in the download dir.

Reply:
in fact i will drop 99999, we are not doing vcs, so only one ebuild for the currently stable release